Vagrantfile

```diff
@@ -0,0 +1,23 @@
+# Encoding: utf-8
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# http://docs.vagrantup.com/v2/
+Vagrant.configure('2') do |config|
+  config.vm.box = 'ubuntu/trusty64'
+  config.vm.hostname = 'url-cpp'
+  config.ssh.forward_agent = true
+
+  # Part of provisioning is cloning a couple of private repos, so this is to
```

Comment:
Comment fail. This is a public repo, so it shouldn't have private dependencies. Also I think the following block can probably be dropped.